resources/js/pages/AdminBugPanel.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+    <div v-if="this.bugs">
+        <h3>{{ $t('admin-bug-panel') }}</h3>
+        <div>
+            <h4>{{ "this is an admin bug panel" }}</h4>
+        </div>
+        <div>
+            <p>
+                sort by:
+                <template v-for="sortable in sortables">
+                    <button v-if="sortable != 'type'" :key="sortable" v-on:click="sort(sortable)">{{sortable}}</button>
+                    <button v-if="sortable == 'type'" :key="sortable" v-on:click="sort(sortable)">{{sortable}}: {{currentSortType}}</button>
```

Comment:
I think you can do with one button, putting the 'v-if' in the text, as that's the only thing that changes. Like 
`{{ sortable == 'type' ? sortable + ': ' + currentSortType : sortable }}`
Something like that, if it works. Could also make it a computed property instead and put the `if` in there.